docs/enhancements/clustersync_data_breakout.md

```diff
@@ -0,0 +1,50 @@
+# Move SyncSet Status out of Etcd
+
+
+## Summary
+The purpose of this enhancement is to break out SyncSet status out of Etcd and move to external storage
+
+## Motivation
+Once a cluster is installed, ClusterSync account for a bulk of Etcd's traffic and storage since it stores per cluster
+status for all matching SyncSets. It can easily be reconstructed if lost, and it is not a resource we reconcile on.
+In order to avoid such regular large writes to Etcd, Hive SyncSet status is the perfect candidate to move out of Etcd.
+This can help Hive clusters scale to the anticipated 1000 per Hive target.
+
+### Goals
+Deploy an alternative storage backend via the Hive Operator and eliminate the ClusterSync Etcd storage.
+
+### Non-Goals
+None
+
+## Open Questions
+1. How to get the details of failing SyncSets for a cluster? Some options to consider:
+    1. Add a small REST API and possibly CLI tool to scrape. However, how to secure the REST API?
+    1. Add a CRD to request SyncSet status, only used when someone is explicitly debugging a problem, and handle cleaning
```

Comment:
What would this CRD look like? Is it Cluster specific or [Selector]SyncSet specific? (probably has to be cluster specific as it is today) When would it be cleaned up? 

What does the flow look like for SRE to know a SyncSet is having problems, and identifying the parameters to make the request?